airflow/models/mappedoperator.py

```diff
@@ -511,6 +510,16 @@ def _get_map_lengths(self, run_id: str, *, session: Session) -> Dict[str, int]:
 
         return map_lengths
 
+    def get_indexes_to_map(self, run_id: str, *, session: Session) -> range:
+        total = 0
+        try:
+            values = self._get_map_lengths(run_id, session=session).values()
```

Comment:
We can only ever be able to expand tasks with literal-only kwargs at this point, so we probably don’t need to call `_get_map_lengths` (which is expensive and accesses db), but instead rely solely on the task kwargs to get this length.